attestation-service/src/lib.rs

```diff
@@ -295,7 +258,8 @@ fn parse_data(
 ) -> Result<(Option<Vec<u8>>, Value)> {
     match data {
         Some(value) => match value {
-            Data::Raw(raw) => Ok((Some(raw), Value::Null)),
+            // Ear RawValue does not support NULL, so use an empty string
+            Data::Raw(raw) => Ok((Some(raw), Value::String("".to_string()))),
```

Comment:
Would this cause ambiguity. As the user might not give input RuntimeData/InitData to AS (null), or the data is an empty string ( `""`). In the former case, verifier will not check the binding of data against the value inside evidence. While for the latter case, the verfier will calculate the digest of `""` and compare it with the claim inside tee evidence.

When we see a `""` value in ear, how could we distinguish it as a non-given runtime data, or a `""` runtime data?

Reply:
This is a bug in the rust-ear crate imo that will hopefully be fixed. I made an issue about it there. https://github.com/veraison/rust-ear/issues/21